src/System.Drawing.Common/tests/Drawing2D/GraphicsPathTest.cs

```diff
@@ -0,0 +1,3294 @@
+﻿//
+// System.Drawing.GraphicsPath unit tests
+//
+// Authors:
+//	Sebastien Pouliot  <sebastien@ximian.com>
+//
+// Copyright (C) 2006-2007 Novell, Inc (http://www.novell.com)
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using Xunit;
+
+namespace Drawing2D
+{
+    public class GraphicsPathTest
+    {
+        private class RunIfFontFamilyGenericMonospaceNotNull : TheoryAttribute
+        {
+            public RunIfFontFamilyGenericMonospaceNotNull()
+            {
+                if (FontFamily.GenericMonospace != null && PlatformDetection.IsNotWindowsNanoServer)
+                {
+                    Skip = "GenericMonospace FontFamily couldn't be found";
+                }
+            }
+        }
+
+        private const float Pi4 = (float)(Math.PI / 4);
+        private const float Delta = 0.0003f;
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Ctor_Default()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                Assert.Equal(FillMode.Alternate, gp.FillMode);
+                AssertEmptyGrahicsPath(gp);
+            }
+        }
+
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [InlineData(FillMode.Alternate, FillMode.Winding)]
```

Comment:
I'm not sure what this InlineData achieves, the function just gets called once, right? If so it might as well be hard coded perhaps?